### DISCUSSION TITLE
繁體化界面

### DISCUSSION THREAD

**Discussion by sos19941015** [hooke007/mpv_PlayKit#122] [mpv 通用]
20220403版的繁體化
大家用看看有什麼問題~~
[portable_config.zip](https://github.com/hooke007/MPV_lazy/files/8668940/portable_config.zip)

目前是不能用右鍵開啟文件

**Comment by hooke007** [comment]
额为什么不在main分支上修改

以及。。。
![Snipaste_2022-05-11_11-24-01](https://user-images.githubusercontent.com/41094733/167828473-22884984-6eaa-48c1-ab10-1d6eff4fd066.jpg)

**Comment by hooke007** [comment]
目前已转向uosc并支持使用自己的任何语言，要测试则查看issue置顶帖
